### DISCUSSION TITLE
Smooth Scroll for Starlight and monolingual English site

### DISCUSSION THREAD

**Discussion by Cirkutry** [mearashadowfax/ScrewFast#166] [Ideas]
- Smooth scroll for the docs would be really nice as the transition from main pages with SS and to the docs with none is sort of jarring.
- Similar to the French monolingual branch, can we have one for English that removes all i18n and components from Navbar and wherever else i18n is seen.
- Also saw some deprecated modules when installing with `npm install` 
![Code_YBylYC4aES](https://github.com/user-attachments/assets/37e51f83-19fd-4f35-a8dd-78ee5173e087)

**Comment by mearashadowfax** [comment]
Hi @Cirkutry!

I've added Lenis to the Docs in the latest update. Regarding the monolingual branch, I’m not sure I understand your request, as the current branch is in English and doesn't include any i18n features. Creating a separate branch for each language doesn’t seem necessary, as developers can easily customize their templates based on their specific needs. 

As for the deprecated packages, we currently don't have direct control over those dependencies, so there's not much we can do immediately. We’ll need to wait for the maintainers to update their packages to remove the deprecated modules. Once updated versions are released, I’ll update the template accordingly.

**Reply by Cirkutry** [reply]
Great, the smooth scroll addition is much appreciated! And thanks for explaining the linguistic side, I'm new to Astro and wrote that suggestion without actually understanding everything properly. I appreciate you taking the time to answer in such detail though! And that deprecated module reasoning makes sense, thanks.

Have a great day!
